lib/tesla_api/stream.rb

```diff
@@ -42,14 +43,14 @@ def stream(&receiver)
       end
     end
 
-    private
-
-    TIMEOUT = 30
-
     def streaming_endpoint
       Async::HTTP::Endpoint.parse(streaming_endpoint_url, alpn_protocols: Async::HTTP::Protocol::HTTP11.names)
     end
 
+    private
+
+    TIMEOUT = 30
+
```

Comment:
This gets `include`d on the `Vehicle` class, so I would rather it stays private.

Reply:
@timdorr yeah - I though it would be okay since it clarified itself as a `streaming_endpoint`.

I refactored it a bit and made both the url and endpoint values static members on the `Stream` module.  I can easily duplicate this code as I'm configuring the proxy, but I'd rather pull the streaming url from the gem in case it ever changes.